gqcnn/training/tf/trainer_tf.py

```diff
@@ -804,17 +813,19 @@ def _save_configs(self):
                       outfile,
                       indent=GeneralConstants.JSON_INDENT)
 
-        # save training script    
-        this_filename = sys.argv[0]
-        out_train_filename = os.path.join(self.model_dir, 'training_script.py')
-        shutil.copyfile(this_filename, out_train_filename)
-
-        # save architecture
-        out_architecture_filename = os.path.join(self.model_dir, 'architecture.json')
-        json.dump(self.cfg['gqcnn']['architecture'],
-                  open(out_architecture_filename, 'w'),
-                  indent=GeneralConstants.JSON_INDENT)
-        
+
+        self.logger.info("TODO: Add save training script back.")
```

Comment:
@richardliaw Thanks for this PR! Did you intend to add back this part of the code? We'd prefer to leave the architecture part as that's needed for the robot runtime system.